SPECS/r8125-module.spec

```diff
@@ -0,0 +1,57 @@
+%define vendor_name Realtek
+%define vendor_label realtek
+%define driver_name r8125
+
+%if %undefined module_dir
+%define module_dir updates
```

Comment:
Following https://xcp-ng.org/docs/develprocess.html#where-the-modules-are-installed-on-the-system this additional module that does not override an existing kernel module should be installed in `extra`, as we did for `dell-rbu-module`